android/BOINC/app/src/main/java/edu/berkeley/boinc/rpc/ProjectRelatedClasses.kt

```diff
@@ -50,7 +52,12 @@ data class ProjectConfig(
                     parcel.readString() ?: "", parcel.readString() ?: "",
                     parcel.readInt(), parcel.readInt(), parcel.readString() ?: "") {
         platforms = arrayListOf<PlatformInfo?>().apply {
-            parcel.readList(this as MutableList<*>, PlatformInfo::class.java.classLoader)
+            if (VERSION.SDK_INT >= VERSION_CODES.TIRAMISU) {
+                parcel.readList(this as MutableList<PlatformInfo>, PlatformInfo::class.java.classLoader, PlatformInfo::class.java)
```

Comment:
This fix produces a warning about unchecked cast, please fix

Reply:
Should be fixed with the latest commit.